resources/cluster-essentials/templates/psp-unprivileged.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: kyma.unprivileged
+  annotations:
+    helm.sh/hook: "pre-upgrade, pre-install"
+    helm.sh/hook-weight: "0"
+    helm.sh/hook-delete-policy: "before-hook-creation"
+spec:
+  allowPrivilegeEscalation: false
+  allowedCapabilities:
+    # Istio-init required
+    - NET_ADMIN
```

Comment:
I'm not sure if we should limit allowed capabilities at this stage. This was not in the scope of the privileged containers issue, so I'm not sure which components might get affected. Not listing allowed capabilities here seems more safe for this iteration.